sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/HiveOrcQuerySuite.scala

```diff
@@ -316,4 +317,33 @@ class HiveOrcQuerySuite extends OrcQueryTest with TestHiveSingleton {
       }
     }
   }
+
+  test("SPARK-32864: Support ORC forced positional evolution") {
```

Comment:
Could you add a partitioned table also because it's the main use case for this option, @peter-toth ?

Reply:
Sure, added in https://github.com/apache/spark/pull/29737/commits/1765c4e6bb1eabd9aa7767212b109f44b50794a7.